client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInterfaces.ts

```diff
@@ -0,0 +1,21 @@
+import { Dispatch, SetStateAction } from 'react';
+
+import InteractedContact from 'models/InteractedContact';
+import FamilyRelationship from 'models/enums/FamilyRelationship';
+import InteractedContactFields from 'models/enums/InteractedContact';
+import { interactedContactsContext } from 'commons/Contexts/InteractedContactsContext';
+
+export interface useContactQuestioningParameters {
+    setFamilyRelationships: Dispatch<SetStateAction<FamilyRelationship[] | undefined>>;
+    setCurrentInteractedContact: Dispatch<SetStateAction<InteractedContact | undefined>>;
+    interactedContactsState: interactedContactsContext;
+};
+
+export interface useContactQuestioningOutcome {
+    getAllRelationships: () => void;
+    saveContact: (interactedContact: InteractedContact) => void;
+    updateInteractedContact: (interactedContact: InteractedContact, fieldToUpdate: InteractedContactFields, value: any) => void;
```

Comment:
Why value any?

Reply:
because it can be any field when updating context